lib/main/atom/views/simpleSelectionView.tsx

```diff
@@ -51,6 +52,7 @@ export async function selectListView<T>({
         didChangeQuery,
         emptyMessage,
       })
+      select.element.classList.add("atom-typescript")
```

Comment:
There is a `itemsClassList` property that's designed for this. Probably a good idea to use that...